lab-materials/lab-env-data/hub-cluster/hub.yml

```diff
@@ -23,23 +23,25 @@ manifests_dir: hubmanifests
 disconnected_operators:
 - name: advanced-cluster-management
   channels:
-  - name: release-2.7
+  - name: release-2.8
 - name: multicluster-engine
   channels:
-  - name: stable-2.2
+  - name: stable-2.3
 - name: topology-aware-lifecycle-manager
   channels:
   - name: stable
 - name: openshift-gitops-operator
   channels:
   - name: latest
+  channels:
+  - name: latest
 - name: lvms-operator
   channels:
   - name: stable-4.13
 disconnected_extra_images:
 - registry.redhat.io/openshift4/ztp-site-generate-rhel8:v4.13
```

Comment:
use a specific tag? like v4.13.1-3

Reply:
fixed. Also included specific channels for gitops (gitops-1.9) and TALM (4.13)